composer.json

```diff
@@ -19,9 +19,16 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0"
+        "php": "^7.4|^8.0",
+        "php-http/httplug": "^2.2",
```

Comment:
I think it worth moving this dependency to require-dev and requiring https://packagist.org/providers/psr/http-client-implementation instead.

Reply:
It's impossible. Sentry needs async http client. 
See https://github.com/yiisoft/yii-sentry/pull/1#discussion_r650594020

Reply:
Can't we depend on [HTTP client PSR](https://www.php-fig.org/psr/psr-18/)? `psr/http-client-implementation`

Reply:
No